pages/events/summer-school.vue

```diff
@@ -0,0 +1,231 @@
+            :elements="week.tableData"
+          />
+        </cv-tab>
+      </cv-tabs>
+    </section>
+
+    <FaqSection class="summer-school-page__section" />
+
+    <AppHelpfulResourcesSection
+      class="summer-school-page__section"
+      :title="helpfulResourcesData.title"
+      :resources="helpfulResourcesData.resources"
+    />
+  </main>
+</template>
+
+<script lang="ts">
+import { Component } from 'vue-property-decorator'
+import QiskitPage from '~/components/logic/QiskitPage.vue'
+import {
+  header,
+  mosaic,
+  agenda,
+  helpfulResources
+} from '~/constants/summerSchool2021Content'
+
+@Component({
+  head () {
+    const title = 'Qiskit Global Summer School 2021'
+    const description = `The Qiskit Global Summer School 2021 is a two-week intensive summer school
+    designed to empower the next generation of quantum researchers and developers with the skills
+    and know-how to explore quantum applications on their own`
+    const image = '/images/events/summer-school/summer-school-logo.png'
+
+    return {
+      title,
+      meta: [
+        {
+          hid: 'twitter:title',
+          name: 'twitter:title',
+          content: title
+        },
+        {
+          hid: 'twitter:description',
+          name: 'twitter:description',
+          content: description
+        },
+        {
+          hid: 'twitter:image',
+          name: 'twitter:image',
+          content: image
+        },
+        {
+          hid: 'twitter:image:alt',
+          name: 'twitter:image:alt',
+          content: title
+        },
+        {
+          hid: 'og:title',
+          property: 'og:title',
+          content: title
+        },
+        {
+          hid: 'og:description',
+          property: 'og:description',
+          content: description
+        },
+        {
+          hid: 'og:image',
+          property: 'og:image',
+          content: image
+        },
+        {
+          hid: 'og:image:secure_url',
+          property: 'og:image:secure_url',
+          content: image
+        },
+        {
+          hid: 'og:image:alt',
+          property: 'og:image:alt',
+          content: title
+        }
+      ]
+    }
+  }
+})
+export default class SummerSchoolPage extends QiskitPage {
+  routeName = 'summer-school'
+  agendaColumnsDataTable: string[] = ['Day', 'Topic', 'Speaker', 'Format']
+  headerData = header
+  mosaicData = mosaic
+  agendaData = agenda
+  helpfulResourcesData = helpfulResources
+}
+</script>
+
+<style lang="scss" scoped>
+.summer-school-page {
+  display: flex;
+  flex-direction: column;
+
+  &__section {
+    @include contained();
+
+    width: 100%;
+    margin-top: $layout-05;
+    margin-bottom: $layout-03;
+
+    @include mq($until: large) {
+      margin-bottom: $layout-01;
+    }
+  }
+}
+</style>
+
+<style lang="scss" scoped>
+// overrides
+.summer-school-page {
+  // stylelint-disable selector-pseudo-element-no-unknown
+  ::v-deep {
+  // stylelint-enable selector-pseudo-element-no-unknown
```

Comment:
This is globally disabled in the Stylelint config in this PR.

```suggestion
  ::v-deep {
```